frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit HPA action
+	As a user, I want to edit the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment 
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user is in topology
+
+@regression
+Scenario: Edit HPA action form view
+   Given user has a workload with HPA assigned to it
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Edit Horizontal Pod Autoscaler option 
+   And user sees Edit Horizontal Pod Autoscaler form opens
+   And user sees values are prefilled
+   And user checks the name value but can not edit it
+   And user checks and edit the value for maximum and minimum pods
+   And user checks and edit the cpu value
+   And user checks and edit the memory value
+   And user clicks on save to get back to topology page
+   And user opens the sidebar
+   And user sees Horizontal Pod Autoscalers section
+   And user goes to the details tab
+   Then user can see the changed pods number
+
+@regression, @manual
+Scenario: Edit HPA action YAML view
+   Given user has a workload with HPA assigned to it
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Edit Horizontal Pod Autoscaler option 
+   And user sees Edit Horizontal Pod Autoscaler page opens with form view selected
+   And user selects YAML view option
+   And user sees schema on the right sidebar
+   And user scrolls the and checks schema
+   And user closes the schema
+   And user checks the YAML
+   And user checks and edits values of cpu and memory under metrics.resource.target.averageUtilization
+   And user checks and edits minimum and maximum pods value in minReplicas and maxReplicas
+   And user checks the name value under metadata.name
+   And user clicks on save to get back to topology page
+   And user opens the sidebar
+   And user can see Horizontal Pod Autoscalers section
+   And user goes to the details tab
+   Then user can see the changed pods number
+
+@regression, @manual
+Scenario: Edit HPA YAML view to form view
+   Given user has a workload with HPA assigned to it
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Edit Horizontal Pod Autoscaler option 
+   And user sees Edit Horizontal Pod Autoscaler page opens with form view selected
+   And user selects YAML view option
+   And user sees schema on the right sidebar
+   And user scrolls the and checks schema
```

Comment:
Can you please update it? I don't understand the statement
Also, is it necessary to add this statement?